PokeOneWeb/Frontend/pokeoneweb/src/environments/environment.ts

```diff
@@ -5,6 +5,7 @@
 export const environment = {
     production: false,
     baseUrl: 'https://localhost:5001'
```

Comment:
Change the base URL on local dev to assets folder for everyone. There's no reason to serve the JSON files using the backend anymore.

Reply:
done in next commit